ddp-workspace/projects/ddp-testboston/src/app/components/header/header.component.html

```diff
@@ -85,6 +95,12 @@
                         Navigation.StayInformed
                     </a>
                 </li>
+                <li class="menu__item">
```

Comment:
Should we remove this from the mobile menu as well since we don't have it in the regular navigation menu anymore?